### PR TITLE
OCP 4.12 : Descheduler Doc updates for P

### PR DESCRIPTION
**Pull request by alishaIBM**
Jira for P/Z : https://issues.redhat.com/browse/MULTIARCH-2638

Draft for P/Z :
https://docs.google.com/document/d/16ccQzO5Ystapc58IGTgtJi3j944e-77zN3Q5dwWQ3sY/edit#

Preview : https://52255--docspreview.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-profiles_nodes-descheduler

Reviewers : 
For P : Paul Bastide
For Z : Muhammad Adeel
For x86 : Rama Kasturi

Branch : enterprise-4.12 and later.

### CODE REVIEW

**Comment by alishaIBM** [comment]
@kasturinarra could you please review for x86 ? Thanks.

/cc @kasturinarra

**Comment by madeelibm** [comment]
lgtm

**Comment by kasturinarra** [comment]
@alishaIBM added lgtm based on my understanding of the PR, but i hope this has been tested atleast once, can you please help confirm ? thanks !!

**Comment by alishaIBM** [comment]
> @alishaIBM added lgtm based on my understanding of the PR, but i hope this has been tested atleast once, can you please help confirm ? thanks !!

@kasturinarra yes it is tested on ppc64le platform.

**Comment by kasturinarra** [comment]
@alishaIBM thanks, then we are good i guess.

**Comment by alishaIBM** [comment]
> @alishaIBM thanks, then we are good i guess.

@kasturinarra okay thank you for the confirmation !

File: modules/nodes-descheduler-profiles.adoc [hunk 1]
```diff
@@ -60,7 +60,7 @@ Do not enable both `SoftTopologyAndDuplicates` and `TopologyAndDuplicates`. Enab
 
 `EvictPodsWithLocalStorage`:: This profile allows pods with local storage to be eligible for eviction.
 
-`EvictPodsWithPVC`:: This profile allows pods with persistent volume claims to be eligible for eviction.
+`EvictPodsWithPVC`:: This profile allows pods with persistent volume claims to be eligible for eviction. When using the `EvictPodsWithPVC`, customers using `Kubernetes NFS Subdir External Provisioner` should add an excluded namespace for the namespace where the provisioner is installed.
```
Comment (snarayan-redhat) [inline modules/nodes-descheduler-profiles.adoc:63]:
```suggestion
`EvictPodsWithPVC`:: This profile allows pods with persistent volume claims to be eligible for eviction.  If you are using `Kubernetes NFS Subdir External Provisioner`, you must add an excluded namespace for the namespace where the provisioner is installed.
```

**Review by snarayan-redhat — COMMENTED** [review]
Looks good! Just one suggestion as per the writing standards.

**Comment by alishaIBM** [comment]
> Looks good! Just one suggestion as per the writing standards.

Thanks @snarayan-redhat for the suggestion. I have updated the file accordingly.

**Pull request merged**